core/src/main/scala/org/typelevel/sbt/TypelevelPlugin.scala

```diff
@@ -17,23 +40,57 @@ object TypelevelPlugin extends AutoPlugin {
   }
 
   import autoImport._
+  import TypelevelKernelPlugin.autoImport._
   import TypelevelSettingsPlugin.autoImport._
   import GenerativePlugin.autoImport._
   import GitHubActionsPlugin.autoImport._
+  import HeaderPlugin.autoImport._
 
   override def globalSettings = Seq(
     tlFatalWarningsInCi := true
   )
 
   override def buildSettings = Seq(
+    organization := "org.typelevel",
+    organizationName := "Typelevel",
+    startYear := Some(java.time.YearMonth.now().getYear()),
```

Comment:
We've got about 364 days until this turns into a bug.

Reply:
:) we can also just put this into the g8 and updated it every year.